docs/virtual_bmc.md

```diff
@@ -39,12 +37,56 @@ How it works
 Supported IPMI commands
 -----------------------
 
-(TBD)
+- Chassis Power On
+- Chassis Power Off
+- Chassis Power Reset / Cycle
 
-HTTPS server
-------------
+Redfish API
+-----------
+
+### Supported Resources
+
+- [ChassisCollection](https://www.dell.com/support/manuals/ja-jp/idrac9-lifecycle-controller-v3.3-series/idrac9_3.36_redfishapiguide/chassiscollection?guid=guid-c4ac8700-44d2-46e9-b90f-67eed0774fce&lang=en-us)
+- [Chassis](https://www.dell.com/support/manuals/ja-jp/idrac9-lifecycle-controller-v3.3-series/idrac9_3.36_redfishapiguide/chassiscollection?guid=guid-c4ac8700-44d2-46e9-b90f-67eed0774fce&lang=en-us)
+  - [Supported Action - Reset](https://www.dell.com/support/manuals/ja-jp/idrac9-lifecycle-controller-v3.3-series/idrac9_3.36_redfishapiguide/supported-action-%E2%80%94-reset?guid=guid-eae5f0af-bfdf-4915-b097-2f6f771e5c08&lang=en-us)
 
-For test purpose, Placemat deploys HTTPS servers on `https://<BMC address>:443`.
-They return 200 OK only.
+Placemat v2 returns the following fixed ChassisCollection.
+
+```json
+{
+  "@odata.context": "/redfish/v1/$metadata#ChassisCollection.ChassisCollection",
+  "@odata.id": "/redfish/v1/Chassis/",
+  "@odata.type": "#ChassisCollection.ChassisCollection",
+  "Description": "Collection of Chassis",
+  "Members": [
+    {
+      "@odata.id": "/redfish/v1/Chassis/System.Embedded.1"
+    }
+  ],
+  "Members@odata.count": 1,
+  "Name": "Chassis Collection"
+}
+```
 
 If you use this feature, prepare certificate files and specify command line options `-bmc-cert` and `-bmc-key`.
```

Comment:
Now that placemat2 provides Redfish API over HTTPS, we should enable
HTTPS even if these flags are not specified.